.travis.yml

```diff
@@ -14,3 +14,8 @@ install:
 script:
   - sed -ri "s/REPO_TOKEN/$REPO_TOKEN/" .coveralls.yml
   - docker-compose exec web bash -l -c "CI=true TRAVIS=true rake test:all"
+
+branches:
```

Comment:
What kind of CI behavior does this chage causes? Does it mean it will only run when certain MR is accepted into `main` branch?

Reply:
It will run for all commits and prs into the main and unstable branch. This prevents the unnecessary push build when creating a pr from a branch in this repo.

Reply:
@kaustubh-nair Sorry, but what I didn't get is it the build happening *before* the MR is accepted into `main` or `unstable` branch or *after*?

Reply:
Hi, the goal here is to try to avoid the following situation when doing a PR:
![Captura de pantalla_2019-04-07_21-30-02](https://user-images.githubusercontent.com/199755/55695129-c22c2300-597c-11e9-81b3-99e14ef33bbc.png)
It's just wasteful to build twice in parallel. 
If I'm not mistaken that is the goal, and a good one! If there's a special case maybe it can be accomodated? Thanks to you all, I'm learning a lot!

Reply:
> Hi, the goal here is to try to avoid the following situation when doing a PR:
> ![Captura de pantalla_2019-04-07_21-30-02](https://user-images.githubusercontent.com/199755/55695129-c22c2300-597c-11e9-81b3-99e14ef33bbc.png)
> It's just wasteful to build twice in parallel.
> If I'm not mistaken that is the goal, and a good one! If there's a special case maybe it can be accomodated? Thanks to you all, I'm learning a lot!

I get it now, so its before merging into `main` or `unstable`. It's a great idea then.